packages/truffle-core/cli.js

```diff
@@ -83,6 +83,14 @@ command.run(inputArguments, options, function(err) {
         process.exit(err);
       } else {
         let error = err.stack || err.message || err.toString();
+        //remove identifying information if error stack is passed to analytics
+        if (error === err.stack) {
+          let directory = __dirname;
+          let identifyingInfo = directory.split("truffle")[0];
```

Comment:
So, theoretically, if the user's username has `truffle` in it, this would cause incorrect results. (`/Users/truffler/.nvm/8.11.0/node_modules/truffle/packages/truffle-core` or somesuch)

But we probably don't need to bother accounting for this. It might be worth a comment though, just to reduce head scratching in 2 years when someone gets bit by this weird case.

Reply:
We could be a bit more specific...like we could just split it by something like `/truffle/packages`

Reply:
@gnidan I will add a comment. I realized this, but also felt like it wasn't worth adding more code to account for that. 
@eggplantzzz I did originally make it more specific but it is not always /truffle/packages. Sometimes it is truffle/build and possibly other iterations as well. This seemed like the most efficient way to catch the vast majority of cases.

Reply:
Oh yeah, just an idea...maybe `/truffle/` is enough to minimize conflicts.

Reply:
@eggplantzzz I looked into this and apparently on some machines the file path uses backslashes instead of forward slashes? Just based on the errors I see in Analytics right now. So I think it's probably better to leave this alone and try to keep it simple if we can. Happy to revisit after we see some more errors in case this solution is woefully inadequate!

Reply:
Oh yeah Windows systems us "\" instead of "/"